bento/crates/workflow/src/tasks/keccak.rs

```diff
@@ -4,13 +4,13 @@
 
 use crate::{
     redis::{self, AsyncCommands},
-    tasks::{serialize_obj, COPROC_CB_PATH},
+    tasks::{serialize_obj, COPROC_CB_PATH, RECEIPT_PATH},
```

Comment:
i suppose it works the same since receipt paths use `uuid`, but i would prefer to either keep `KECCAK_RECEIPT_PATH` or siwtch to `RECUR_RECEIPT_PATH`